### ISSUE TITLE
Expecting keyfile to have an 'address' element

### ISSUE THREAD

**Issue by carver** [pipermerriam/ethereum-keyfile#2]
web3.js v1 claims to [produce an `'address'` element when encrypting a json keyfile](https://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html#encrypt). (The public account address, as you'd expect)

Can that be added in `ethereum-keyfile`?

Everything else looks like a match (except that web3.js defaults to scrypt, but I don't see why that needs to be the same).

**Comment by pipermerriam** [comment]
Yes, it was left out originally because I needed the ECDSA operations to convert the private key to the address, and I hadn't done those yet.  Should be a very simple addition.

**Comment by pipermerriam** [comment]
Any chance you're willing to add it?

**Comment by carver** [comment]
Sure, I can set it up tomorrow

**Comment by stefanmendoza** [comment]
@carver was looking through the issues to find something to work on and I think this one can be closed with #4. 👍
